proto/vulcanize/auction/v1beta1/query.proto

```diff
@@ -117,7 +129,12 @@ service Query {
 
     // AuctionsByBidder queries auctions by bidder
     rpc AuctionsByBidder(AuctionsByBidderRequest) returns (AuctionsByBidderResponse) {
-        option (google.api.http).get = "/vulcanize/auction/v1beta1/auctions/{bidder_address}";
+        option (google.api.http).get = "/vulcanize/auction/v1beta1/auctions-by-bidder/{bidder_address}";
```

Comment:
May be we can just use by-owner instead of auctions-by-owner as it is in the same module.